project/ProjectPlugin.scala

```diff
@@ -34,11 +34,13 @@ object ProjectPlugin extends AutoPlugin {
     val clientSettings: Seq[Def.Setting[_]] = Seq(
       libraryDependencies ++= Seq(
         %%("cats-core", V.cats),
-        "io.higherkindness"               %% "compendium-common" % V.compendium,
-        "com.pepegar"                     %% "hammock-core" % V.hammock,
-        "com.pepegar"                     %% "hammock-circe" % V.hammock,
-        "com.pepegar"                     %% "hammock-asynchttpclient" % V.hammock,
+        "io.higherkindness" %% "compendium-common"       % V.compendium,
+        "com.pepegar"       %% "hammock-core"            % V.hammock,
+        "com.pepegar"       %% "hammock-circe"           % V.hammock,
+        "com.pepegar"       %% "hammock-asynchttpclient" % V.hammock,
+        //"com.pepegar"                     %% "hammock-apache-http" % V.hammock,
         "com.beachape"                    %% "enumeratum" % V.enumeratum,
+        "com.julianpeeters"               %% "avrohugger-core" % "1.0.0-RC22",
```

Comment:
We should put the avrohugger version under `V` object, with the rest of the lib versions